# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/python-maturin/hotfix-cargo-locked.patch.ia64
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- maturin-1.14.1/setup.py.vanilla	2026-08-01 16:02:07.890506415 +0200
+++ maturin-1.14.1/setup.py	2026-08-01 16:02:20.650919558 +0200
@@ -60,6 +60,6 @@
 setup(
     version=version,
     cmdclass={"bdist_wheel": bdist_wheel},
-    rust_extensions=[RustBin("maturin", args=cargo_args, cargo_manifest_args=["--locked"], env=env)],
+    rust_extensions=[RustBin("maturin", args=cargo_args, env=env)],
     zip_safe=False,
 )
